charts/refinery/ci/debug.yaml

```diff
@@ -0,0 +1,11 @@
+resources:
+  limits:
+    cpu: 100m
+    memory: 200M
+  requests:
+    cpu: 100m
+    memory: 200M
+
+debug:
+  enabled: true
+  port: 6062
```

Comment:
Should this be 6061 to match values.yaml?

Reply:
No, this test is specifically showing that the value can be non-default and the chart will render.  I do need to update the default value to be 6060 tho.